test-load/pkg/internal/sets/string.go

```diff
@@ -0,0 +1,153 @@
+package sets
```

Comment:
why build the set? https://pkg.go.dev/github.com/golang-collections/collections/set#pkg-types